vaccine_feed_ingest/runners/il/sfsites/normalize.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+import pydantic
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+URL_RE = re.compile(
+    r"(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})"
+)
+
+
+def _get_address(site: dict) -> schema.Address:
+    street1 = site["Street__c"]
+    city = site["City__c"]
+    zipc = site["Postal_Code__c"]
+
+    if not schema.ZIPCODE_RE.match(zipc):
+        zipc = None
+
+    return schema.Address(
+        street1=street1,
+        street2=None,
+        city=city,
+        state="IL",
```

Comment:
Where possible (in the future), please [use the enums](https://github.com/CAVaccineInventory/vaccine-feed-ingest-schema/blob/b342937b944ef477fee11512448be39181c1bfb5/vaccine_feed_ingest_schema/location.py#L108). You do a great job of this below!
```suggestion
        state="schema.State.ILLINOIS",
```